composer/callbacks/global_straggler_detector.py

```diff
@@ -0,0 +1,873 @@
+from megatron.core.utils import *
+
+from composer.core import Callback, State, Event, Time
+from composer.loggers import Logger
+from composer.utils import dist
+from composer.models.base import ComposerModel
```

Comment:
we generally group composer imports separately after the first section of generic imports (see other files). It makes tracking internal vs. external imports easier